src/Common/src/Interop/Unix/libcurl/Interop.libcurl_types.cs

```diff
@@ -139,6 +163,16 @@ internal struct CURLMsg
             internal int result;
         }
 
+        // Curl Version Information
+        [StructLayout(LayoutKind.Explicit)]
+        internal struct curl_version_info_data
+        {
+            [FieldOffset(0)]
+            internal int age;
+            [FieldOffset(32)]
+            internal int features;
+        }
```

Comment:
I see.  In that case I think it'd be better to use the default of SequentialLayout and include all the fields up through the one you care about, in particular because pointer sizes can change based on 32-bit/64-bit.  Now, with the work @nguerrera's pioneering to introduce native shims, this will end up being in native anyway, and we'll only surface the information we care about, but in the meantime, I think it'd be good to see it something like:
```C#
// Note that this struct is larger than is what's defined here.
// This definition is only valid for use with Marshal.PtrToStructure
// and not for general use in P/Invoke signatures.
internal struct curl_version_info_data
{
    internal CURLversion age;
    private unsafe char* version;
    private int version_num;
    private unsafe char* host;
    internal int features;
    // ... ommitting additional fields here
}
```